listView.php

```diff
@@ -5,37 +5,90 @@
 <head>
     <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@4.5.3/dist/css/bootstrap.min.css" integrity="sha384-TX8t27EcRE3e/ihU7zmQxVncDAy5uIKz4rEkgIXeMed4M0jlfIDPvg6uqKI2xXr2" crossorigin="anonymous">
     <title>Student Planner</title>
+</head>
 
     <body>
         <h1 class="text-center">Student Planner</h1>
 	<br>
         <h2 class="text-center">List View</h2>
 	<br>
 
-    </body>
+    <button type="button" class="btn btn-primary" data-toggle="modal" data-target="#createTask">
+  Create Task
+</button>
 <?php
 require("conn.php"); 
 
+if (!empty($_POST['name'])){
+	//If the user submitted a new list request we run this code
```

Comment:
For this, I tested the following:

- Creating a list with an 'empty' input(a string of spaces): the list was still created, with no name. I'm not sure whether this should be fixed since it technically isn't "empty", but will make it difficult to access/assign tasks to.

- Creating a list with an existing name: a list with the same name as an existing was created, but the tasks weren't duplicated, because of it having a different list ID. For just displaying lists, this won't be an issue, but when assigning tasks, it will, since we plan to assign tasks to lists just based on their name, and not their unique ids. 

For the duplicate name case, I was thinking that either an error message can be displayed, or the names can have some sort of extensions(e.g. "_1" or "(1)") attached to them.